.github/workflows/reusable_upload_web.yml

```diff
@@ -151,10 +151,12 @@ jobs:
 
             - [ ] I have tested the web viewer
 
-            | Result | Commit  | Link  |
-            | ------ | ------- | ----- |
+            | Result | Commit  | Link |
+            | ------ | ------- | ---- |
             | ✅ | ${{ steps.get-sha.outputs.sha }} | https://rerun.io/viewer/pr/${{ github.event.pull_request.number }} |
```

Comment:
```suggestion
            | Result | Commit  | Link | Manifest |
            | ------ | ------- | ---- | -------- |
            | ✅ | ${{ steps.get-sha.outputs.sha }} | https://rerun.io/viewer/pr/${{ github.event.pull_request.number }} | [+nightly](https://rerun.io/viewer/pr/${{ github.event.pull_request.number }}?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json) [+main](https://rerun.io/viewer/pr/${{ github.event.pull_request.number }}?manifest_url=https://app.rerun.io/version/main/examples_manifest.json) |
```

Reply:
(perhaps the base URL can also constructed in a separate step to make this markup shorter)